documentation/assemblies/metrics/assembly_metrics-prometheus-setup.adoc

```diff
@@ -7,12 +7,18 @@
 
 This section describes how to monitor Strimzi Kafka, ZooKeeper, Kafka Connect, and Kafka MirrorMaker and MirrorMaker 2.0 clusters using Prometheus to provide monitoring data for example Grafana dashboards.
 
-Prometheus and Grafana can also used to monitor Strimzi operators.
+Prometheus and Grafana can also be used to monitor Strimzi operators and Kafka Bridge instances.
```

Comment:
Maybe make the _Kafka Bridge instances_ a link to the Kafka Bridge section and move the bullet section to the Kafka Bridge section introduction, as we're not setting up here.

Reply:
Which bullet section do you mean?